examples/features/customloadbalancer/client/main.go

```diff
@@ -16,6 +16,7 @@
  *
  */
 
+// Binary client is an example client for custom load balancer.
```

Comment:
This is not proper English.  "for _a_ custom load balancer" or "a client for the custom load balancer example", etc.

Reply:
Thanks. Done